OWNERS_ALIASES

```diff
@@ -9,6 +9,11 @@ aliases:
     - mmitoraj
     - majakurcius
     - alexandra-simeonova
+  technical-writers:
+    - klaudiagrz
+    - mmitoraj
+    - majakurcius
+    - alexandra-simeonova
```

Comment:
```suggestion
    - NHingerl
```
@alexandra-simeonova has moved to another unit with the Luigi team, additionally, @NHingerl is missing from the list

Reply:
@majakurcius please make commit that reflect changes for technical-writers, as per instruction in PR body.